pkgs/development/python-modules/qscintilla-qt5/default.nix

```diff
@@ -27,6 +27,8 @@ buildPythonPackage {
     lndir ${pyqt5} $out
     rm -rf "$out/nix-support"
     cd Python
+    sed -i configure.py \
+      -e "s/qmake = {'CONFIG': 'qscintilla2'}/qmake = {'CONFIG': 'qscintilla2', 'QT': 'widgets', 'QT': 'printsupport'}/"
```

Comment:
I have not gone too deeply, but as far as I can see, this python code will evaluate `qmake` variable to be `{'CONFIG': 'qscintilla2', 'QT': 'printsupport'}` (widgets will be dropped).

I have not been into this `configure.py` deeply, but I guess what you want here is

```python
qmake = {
    'CONFIG': 'qscintilla2',
    'QT': 'widgets printsupport'
}
```

Reply:
Yes, you are right; changed this now. I had not looked at the .pro file to see if the script does what I supposed it does, otherwise I would have noticed :)

Reply:
No problem.

Also, this seams to be a upstream issue. I have no problem having this kind of fix here, but it would make sens to raise the issue upstream. Have you done that already?

Reply:
No, I haven't. Riverbank Computing, the developer, don't make it particularly easy to contribute. As far as I can see, they only host their code on their on webpage.